conda_devenv/devenv.py

```diff
@@ -456,10 +456,12 @@ def get_env_directory(env_name):
     import json
     info = subprocess.check_output(["conda", "info", "--json"]).decode()
     info = json.loads(info)
-    envs = info["envs"]
+    envs_dirs = info["envs_dirs"]
 
-    for env in envs:
-        if os.path.basename(env) == env_name:
+    for directory in envs_dirs:
+        env = os.path.join(directory, env_name)
+        conda_meta_dir = os.path.join(env, 'conda-meta')
+        if os.path.isdir(conda_meta_dir) and os.path.basename(env) == env_name:
             return env
 
     return None
```

Comment:
This ``return None`` is not necessary right
:)

Reply:
Hmm what do you mean?

Reply:
Oh that by default all functions return `None`?

You are right that it is not necessary, but particularly I prefer to `return None` explicitly from functions which return something.

Reply:
Exactly! No problem! :)